src/app/public/modules/repeater/repeater.component.ts

```diff
@@ -73,6 +73,13 @@ export class SkyRepeaterComponent implements AfterContentInit, OnChanges, OnDest
   @Output()
   public activeIndexChange = new EventEmitter<number>();
 
+  /**
+   * Fires when the order of repeater items change.
+   * The event emits an ordered array of `tag`s based on the `tag` property for each repeater item that the consumer provides.
```

Comment:
Should this be pulled up to the previous line? Looking at the JSDocs for popovers, it looks like stuff is on the same line unless we want it to be a separate paragraph. For the `tab` input, it makes sense to have the required info as a separate paragraph because we want it to stand out. but here, the content on line 78 just seems like a continuation of the description that should be in the same paragraph.

Reply:
Also, what do you think about reordering that sentence to tighten up a bit? I'm thinking: "This event emits an ordered array of the `tag` properties that the consumer provides for each repeater item."

Reply:
I reordered the sentence as you suggested. However, the multiple lines are there to avoid the linting rule we have in place that each line can't exceed 140 characters. I double checked and our docs tools do not pick up on line breaks. So, even though this description is on multiple lines, it will still appear as one line in the docs.